### PR TITLE
Fix Minerva and myCourses login

### LINKED ISSUE #116: [BUG] Cannot login due to bad login detection mechanism
This part checks for string "WELCOME" on response:

https://github.com/jguerinet/MyMartlet/blob/31c542a167e1810fff20c943ce36e4039075aaaa/app/src/main/java/util/manager/McGillManager.kt#L135

However, the current Minerva behaviour is that the response of `twbkwbis.P_ValLogin` is *empty* if correct credentials are entered (Incorrect credential does return HTML)

### PR DESCRIPTION
**Pull request by austinhuang0131**
* Fixes #116. In the original issue I said it's "empty" but apparently there's a `<head>` refresh tag so we'll use that
* Also made myCourses login relatively easier (by putting in the username and password) but [still requires the new Microsoft 2FA](https://www.mcgill.ca/2fa/)... and it's mandatory starting next year, so that's that
* Removed `jcenter()` because jcenter deprecation

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,11 +21,12 @@ buildscript {
     ext.kotlin_version = '1.4.0'
     repositories {
         google()
-        jcenter()
+        mavenCentral()
```
Comment (jguerinet) [inline build.gradle:24]: Have you checked that all of the dependencies previously available on `jcenter` are now on `mavenCentral`? I know of a few deps that haven't been migrated yet, so I want to ensure they are all available before making the switch here.

File: build.gradle [hunk 2]
```diff
@@ -21,11 +21,12 @@ buildscript {
...
+        gradlePluginPortal()
```
Comment (jguerinet) [inline build.gradle:25]: Why do we need this? Are there plugins not resolving?

**Review by jguerinet — COMMENTED** [review]
Good PR. Left 2 questions for you, but looks good.

File: build.gradle (hunk 2 above)
Comment (austinhuang0131) [reply build.gradle:25]: It's just for https://github.com/ben-manes/gradle-versions-plugin

File: build.gradle (hunk 1 above)
Comment (austinhuang0131) [reply build.gradle:24]: See the other comment, everything else works

**Pull request merged**